### ISSUE TITLE
Bug: qtfm don't show remove dialog confirmation for symbolic link

### ISSUE THREAD

**Issue by ferion11** [rodlie/qtfm#146]
Bug: qtfm don't show remove dialog confirmation for symbolic link.

**Comment by ferion11** [comment]
If, in your selection of many files, there is symbolic links between normal files, and you press Delete key, it will show the remove dialog confirmation, but it will remove all symbolic links without confirmation first. And if you move a symbolic link, qtfm make a copy.

**Comment by rodlie** [comment]
This is on master (6.3)? I replaced the copy/move function in master and I might have overlooked something. Will check, I'm occupied for a couple of days, but will then dedicated a day to qtfm :)

**Comment by ferion11** [comment]
Yes, is in master (6.3). It isn't a big problem for me, but is better to fix before release the 6.3 version. Take your time :)

**Comment by rodlie** [comment]
Will of course be fixed before release :)

**Comment by ferion11** [comment]
We have some suspicious code here:
https://github.com/rodlie/qtfm/blob/774b9674a325984085c7c071857e1a8cb354ab3d/fm/src/actiontriggers.cpp#L315-L338

**Comment by ferion11** [comment]
There is also the fact that broken symbolic links don't show in qtfm (master 6.3). But I will see the move thing first.

**Comment by ferion11** [comment]
The https://github.com/rodlie/qtfm/pull/148 fix the broken symbolic link thing. Now, for this bug report, the only thing that I remember is the move symbolic link action that don't remove the original.
